app/models/game.rb

```diff
@@ -2,4 +2,9 @@ class Game < ActiveRecord::Base
   belongs_to :white_user, :class_name => 'User'
   belongs_to :black_user, :class_name => 'User'
   has_many :pieces
+
+  scope :needing_second_player, -> { where(black_user_id: nil) }
```

Comment:
I'm about 80% sure just "black_user" would work as Rails would translate that into "black_user_id" but this definitely works since it's explicit.